healthyregions/SDOHPlace
# #141 and #142

This pull request addresses #141 and #147 but is in the initial stage and could change direction based on further design decisions.

### How to Test
1. Open the search page. You should see the Spatial Resolution filter removed from the filter list and now at the top of the left panel.

2. During the initialization stage, only state and county layers have been applied to the map. (The blue dots are my test on spatial coverage locations. They are the counties of the City Health Dashboard. I leave the code in case we need this in the future, but they don't belong to #141 and #147).

3. Play around with the Spatial Resolution filter. You should see the layers change based on your selection, but the state and county layers are always there.

4. Perform other random search and filter actions. The Spatial Resolution layer should not be affected by these operations. (i.e., it is a distinct control of the map)

NOTE:

1. I think this matches our plan for the map behavior, but please let me know if not, and I will modify the function accordingly. I think I should probably end here for now and resume after we make more decisions on map behavior.
2. There is a lot of commented-out code and some code that is currently not used. I still keep them there because there's a probability we will need them in the future, as we are still in the decision-making process for the map.
